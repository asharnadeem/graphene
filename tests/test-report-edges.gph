int main()
{
    node<string> m, n;
    m <->["hello"] n;
    
    list<edge<string>> l = m.edges;
    
    print(m.edges[0].weight);
}