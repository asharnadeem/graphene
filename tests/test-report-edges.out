<<<<<<< HEAD
hello
=======
5
Traversable
>>>>>>> 648036fc5630173fed1923a43cd873a6fc573761
